spec/mfa/hotp.spec.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+describe("HOTP", () => {
+    var hotp, twofaMock, promiseMock;
+
+    beforeEach(() => {
+        var HOTP;
```

Comment:
`Hotp` instead of `HOTP` if it is a class, `hotp` for an instance or a dumb object.